setup.py

```diff
@@ -291,7 +296,7 @@ def run(self):
                 target_dir = os.path.join(SCRIPT_DIR, "build", d)
                 rmtree(target_dir, ignore_errors=True)
                 copytree(os.path.join("build", d), target_dir)
-        os.chdir(ORIG_DIR)
+        os.chdir(os.getcwd())
```

Comment:
This doesn't seem to do anything, does it?

Reply:
Yes, this is still the debug code. I think the path generating wheel seems changed with my modification.

Reply:
Right, it's going to `build/py3.6-64bit/python/dist/` rather than the default `dist/`. The option to change it is `--dist-dir`, but it doesn't seem to be used anywhere here..